pkg/sources/adapter/azureeventhubsource/adapter.go

```diff
@@ -146,28 +149,50 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 
 // Start implements adapter.Adapter.
 func (a *adapter) Start(ctx context.Context) error {
+	go health.Start(ctx)
+
 	connCtx, cancel := context.WithTimeout(ctx, connTimeout)
 	runtimeInfo, err := a.ehClient.GetRuntimeInformation(connCtx)
 	cancel()
 	if err != nil {
-		a.logger.Fatal("GetRuntimeInformation failed:", err)
+		return fmt.Errorf("getting Event Hub runtime information: %w", err)
 	}
 	a.runtimeInfo = runtimeInfo
 
+	a.logger.Info("Starting Event Hub message receivers for partitions ", runtimeInfo.PartitionIDs)
```

Comment:
(Question not a review) should this be `Infof()`?

Reply:
Infof would assume there are formatting markers in the string (`%s`, etc.), but there are none here.